client/cli/src/commands/runcmd.rs

```diff
@@ -292,6 +283,10 @@ impl CliConfiguration for RunCmd {
 		Some(&self.keystore_params)
 	}
 
+	fn offchain_worker_params(&self) -> Option<&OffchainWorkerParams> {
+		Some(&self.offchain_worker_params)
+	}
```

Comment:
Don't use Option here if you don't need. The method name needs to be identical but not the signature.

cc @bkchr

Reply:
ah wait, that's the trait. Nevermind!! :sweat_smile:

Reply:
whoops, then I'll undo that last commit :)